### ISSUE TITLE
Episode 8 - Collaborating: Update output

### ISSUE THREAD

**Issue by kekoziar** [swcarpentry/git-novice#626]
September 2018 the git base code was updated to add an "Enumerating objects" line for some output. So, 
~~~
Enumerating objects: 4, done.
~~~
and 
~~~
remote: Enumerating objects: 4, done.
~~~
need to be added at the beginning of code output for `git push origin master` and `git pull origin master`, respectively.

**Comment by kekoziar** [comment]
`git pull origin master` output is
~~~
remote: Enumerating objects: 4, done.
remote: Counting objects: 100% (4/4), done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 3 (delta 0), reused 3 (delta 0), pack-reused 0
Unpacking objects: 100% (3/3), done.
From https://github.com/vlad/planets
 * branch            master     -> FETCH_HEAD
   9272da5..29aba7c  master     -> origin/master
Updating 9272da5..29aba7c
Fast-forward
 pluto.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 pluto.txt
~~~

**Comment by munkm** [comment]
I think this should also be accompanied by a note in the instructor notes that learners may have slightly different output if they're using an older version of git.

**Comment by munkm** [comment]
Closed via swcarpentry/git-novice#656
